docs/content/community/bijeenkomsten/20181004-demo-sprint-4.md

```diff
@@ -6,97 +6,81 @@ Den Haag, 4 oktober 2018
 
 NB: Mogelijk zou dit in één gecombineerde "convenience method" opgenomen kunnen worden.
 
-#### Wanneer een document bij een besluit hoort, moet dit dan zowel aan de zaak als het besluit gekoppeld worden? Deze opzet betekent dat de componenten (ZRC en DRC) elkaar moeten kunnen bereiken?
-
-Dat klopt inderdaad. Mogelijk spelen hier ook nog authenticatie en autorisatie-uitdagingen mee, dit zullen we moeten bekijken. Autorisatie en Authenticatie staat voor de komende sprint (5) op het programma.
-
-#### Moet deze invulling extra eisen stellen aan het compliant zijn?
+### Wanneer een document bij een besluit hoort, moet dit dan zowel aan de zaak als het besluit gekoppeld worden? Deze opzet betekent dat de componenten (ZRC en DRC) elkaar moeten kunnen bereiken?
+Dat klopt inderdaad. Mogelijk spelen hier ook nog authenticatie en autorisatie-uitdagingen mee, dit zullen we moeten bekijken. Autorisatie en Authenticatie staat voor de komende sprints (5, 6, 7) op het programma.
 
-Dat klopt inderdaad, waar voorheen compliancy werd vastgesteld op berichtuitwisseling worden nu ook eisen gesteld aan de architectuur en hoe de berichten verwerkt worden. Dit is een bekend probleem met microservices wat opgelost kan worden met NLX. Theo Peters (VNG) concludeert dat een applicatie niet alleen moet voldoen aan de API maar ook aan gebruik van NLX.
-
-#### Conflicteert het dubbel vastleggen van de relatie niet met het CG principe van eenmalige opslag?
+### Moet deze invulling extra eisen stellen aan het compliant zijn?
+Dat klopt inderdaad, waar voorheen compliancy werd vastgesteld op berichtuitwisseling worden nu ook eisen gesteld aan de architectuur en hoe de berichten verwerkt worden. Dit is een bekend probleem met microservices wat opgelost kan worden met NLX. Er wordt geconcludeerd dat een applicatie niet alleen moet voldoen aan de API maar ook aan gebruik van NLX.
 
+### Conflicteert het dubbel vastleggen van de relatie niet met het CG principe van eenmalige opslag?
 Nee, het informatieobject wordt nog steeds maar eenmaal opgelagen. Alleen de relatie met andere objecten wordt meermalen opgeslagen, dit zjn echter verwijzingen.
 
-#### Hoe borgen we dat relaties intact blijven? Bijvoorbeeld bij het migreren van applicaties/databases? Of het samenvoegen van meerdere DRC's tot één groot DRC of juist het opslitsen van een groot DRC in meerdere kleinere?
-
+### Hoe borgen we dat relaties intact blijven? Bijvoorbeeld bij het migreren van applicaties/databases? Of het samenvoegen van meerdere DRC's tot één groot DRC of juist het opslitsen van een groot DRC in meerdere kleinere?
 Door het accepteren van eventual-consistency beseffen we ook dat dit door verschillende
 factoren uit elkaar kan gaan lopen. We bedachten toen ook dat het technisch mogelijk
 is om periodieke, automatische checks te laten draaien die consistentie controleren
 en zelfs kunnen oplossen.
 
-#### Is de DRC is de master en kan daar altjd de juiste verwijzing opgezocht worden?
-
```

Comment:
Deze newlines staan er met een reden - is markdown style.

Reply:
sorry moet er ook vanaf blijven, moet ik het handmatig terugzetten? of kan ik jouw suggesties gemakkelijk en automatisch overnemen?

Reply:
Ik doe het even, gaat sneller in mijn code editor ;)